sillsdev/web-languageforge
# Chore/1548 feat home page improvements

Fixes #1548

## Description

1) Made some visual improvements for our homepage
2) Added some basic E2E tests for our homepage

I'm not strongly attached to any of the images I picked, so I'm open to suggestions. All 3 of our images have a similar temperature now, which I like.

My style change in main.css is a bit hard to spot, so here's a screenshot. I only used colour-codes that are already used elsewhere in the theme:
![Screenshot_20230109_045406](https://user-images.githubusercontent.com/12587509/211569234-065ce175-4414-4196-9767-a7572c04446f.png)

### Type of Change

- UI change

## Screenshots

![Screenshot_20230109_045420](https://user-images.githubusercontent.com/12587509/211569073-1c18d9d2-ae70-4912-87ee-0db105e452ff.png)
![Screenshot_20230109_045338](https://user-images.githubusercontent.com/12587509/211569111-36f645eb-6a1a-4865-8e98-d5c7b20e0fee.png)

## Checklist


## How to test

- Make sure images load and are positioned properly and the change in `main.css` didn't unexpectedly make anything yucky.
- For the new E2E tests: Let 'em rip 🔥

## qa.languageforge.org testing

Testers should add his/her findings to end of the PR in a comment and include screenshots, files, etc that are beneficial.
